CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKVisualProcessor.java

```diff
@@ -68,19 +68,25 @@
     
     private final BlockingQueue<CVKRenderable.CVKRenderableUpdateTask> taskQueue = new LinkedBlockingQueue<>();
     private final CVKCanvas cvkCanvas;
-    private final CVKRenderer cvkRenderer;
     private final Frustum viewFrustum = new Frustum();
     private final Matrix44f projectionMatrix = new Matrix44f();       
-    private CVKIconTextureAtlas cvkIconTextureAtlas = null;
-    private CVKAxesRenderable cvkAxes = null;
-    private CVKFPSRenderable cvkFPS = null;
-    private CVKIconsRenderable cvkIcons = null;
-    private CVKPointsRenderable cvkPoints = null;
+    private final CVKIconTextureAtlas cvkIconTextureAtlas;
```

Comment:
how come the default values were removed?

Reply:
Final.